scripts/test-all

```diff
@@ -9,17 +9,17 @@ set -euo pipefail
 # Run from the Makefile environment
 MAKEFILE_RUN=${MAKEFILE_RUN:-""}
 if [[ -z $MAKEFILE_RUN ]] ; then
-  COMMAND="$0 $*" exec make run
+    COMMAND="$0 $*" exec make run
 fi
 
-./scripts/test "$@" -- --nocapture
+./scripts/test "$@"  && echo
```

Comment:
Why remove `--nocapture`?

Reply:
See the document, nextest is not consistent with cargo on test binary arguments.
```
# Some arguments are different in nextest.
./scripts/test $TESTNAME --no-capture
```

Reply:
Better keep the old behavior.